crates/red_knot_python_semantic/resources/mdtest/annotations/starred_expressions.md

```diff
@@ -0,0 +1,18 @@
+# Starred expression annotations
+
+Type annotations for `*args` can be starred expressions themselves:
+
+```py
+from typing import TypeVarTuple
```

Comment:
nit: this should future-proof the test a little bit. Strictly speaking we should complain about an import of `typing.TypeVarTuple` with our default Python version, since `typing.TypeVarTuple` was added in Python 3.11. `typing_extensions.TypeVarTuple`, on the other hand, is available on all Python versions.

```suggestion
from typing_extensions import TypeVarTuple
```